sdcm/sct_config.py

```diff
@@ -1124,7 +1124,29 @@ class SCTConfiguration(dict):
              help="Limit severity level for event types"),
 
         dict(name="scylla_rsyslog_setup", env="SCT_SCYLLA_RSYSLOG_SETUP", type=boolean,
-             help="Configure rsyslog on Scylla nodes to send logs to monitoring nodes")
+             help="Configure rsyslog on Scylla nodes to send logs to monitoring nodes"),
+
+        dict(name="data_device", env="SCT_DATA_DEVICE", type=str,
+             help="""Define which device to use for scylla data. Default for gce is attached. Default for aws is instance_store.
+                     attached - means use additionaly attached devices
+                     instance_store - means use emphereal local NVMes devices(AWS)""",
+             choices=("attached", "instance_store")),
+
+        dict(name="aws_ebs_volume_num", env="SCT_AWS_EBS_VOLUME_NUM",
+             type=int,
+             help="Number of additional ebs volumes attached to instances"),
+
+        dict(name="aws_ebs_volume_type", env="SCT_AWS_EBS_VOLUME_IOPS",
+             type=str,
+             help="Type of addtitional volumes: gp2|gp3|io2|io3"),
+
+        dict(name="aws_ebs_volume_size", env="SCT_EBS_VOLUME_SIZE",
+             type=int,
+             help="Size of additional volume in GB"),
+
+        dict(name="aws_ebs_volume_iops", env="SCT_EBS_VOLUME_IOPS",
+             type=int,
+             help="Number of iops for ebs type io2|io3|gp3"),
```

Comment:
Please, fix names of env vars (see comment for .md)

Reply:
fixed